_pages/how-we-work/glossary.md

```diff
@@ -49,25 +53,29 @@ tags:
 
 * **Grok** - To fully [understand](http://www.urbandictionary.com/define.php?term=grok) something.
 
+* **HR Links** - [HR Links](https://corporateapps.gsa.gov/hr-links/) is GSA's system for HR actions, including requesting leave and managing performance review documents.
+
 * **IAA** - Interagency agreement. A contract between federal agencies.
 
 * **MOA** - The [Memorandum of agreement](https://docs.google.com/a/gsa.gov/file/d/0B84F26FpUP0lQU5aWkplWDVtS2M/edit), created between GSA and FAS establishes us through 2018.
 
 * **MOU** - Memorandum of understanding.
 
-* **OLU** - On-Line University, where our team's mandatory training classes are hosted.
+* **OLU** - On-Line University, where our agency's mandatory training classes are hosted.
+
+* **OPP** - The [Office of Products and Programs](../office-of-products-and-programs/) helps the public use and understand government, and helps agencies understand and serve the public. To achieve this, they create, discover, connect and share practical solutions and products that transform government. OPP is a division within TTS.
 
 * **OTP** - One-time password. In Atlanta, this refers to "outside the perimeter," as opposed to ITP (inside the perimeter).
 
-* **PIF** - Presidential Innovation Fellows serve on highly competitive, one-year appointments, which fall within 18F.
+* **PIF** - The [Presidential Innovation Fellows program](https://presidentialinnovationfellows.gov/) pairs talented, diverse technologists and innovators with top civil-servants and change-makers working at the highest levels of the federal government to tackle some of the nation’s biggest challenges. The PIF program is highly-competitive, and PIFs serve on one-year appointments. The PIF team is a division within TTS.
```

Comment:
This feels a bit superlative-driven, especially in comparison to the entries for 18F, USDS and CoEs. Not to diminish the fine work the PIFs are doing, but perhaps it could be toned down a bit?